src/Mentat.Repository/Models/Card.cs

```diff
@@ -8,6 +8,16 @@ namespace Mentat.UI.Models
     public class Card
     {
         public Object _id { get; set; }
+        public string subject { get; set; }
+
+        public string question { get; set; }
+
+        public string answer { get; set; }
+
+        public bool isCustom { get; set; }
+
+        public string difficulty_level { get; set; }
+
         public string notes { get; set; }
     }
```

Comment:
Needs pascal casing. Typically you shouldn't snake case in C# unless its a serialization requirement to be compatible with something else.